modules/ssh/ssh.go

```diff
@@ -15,13 +15,15 @@ import (
 
 	"github.com/gruntwork-io/terratest/modules/logger"
 	"golang.org/x/crypto/ssh"
+	"golang.org/x/crypto/ssh/agent"
 )
 
 // Host is a host on AWS.
 type Host struct {
 	Hostname    string
 	SshUserName string
 	SshKeyPair  *KeyPair
+	SshAgent    bool
```

Comment:
Now that there are a few options here, could you add a comment to the right of each field? i.e., mention what `SshKeyPair` are and that one or both can be used.

Reply:
Agreed.